providers/kavenegar-novu-sms/src/lib/kavenegar-novu-sms.provider.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { KavenegarNovuSmsSmsProvider } from './kavenegar-novu-sms.provider';
+
+test('should trigger kavenegar library correctly', async () => {
```

Comment:
By using Jest as test runner and Axios as HTTP client provider you would need to set this up in the Jest config to make it work properly:
https://github.com/novuhq/novu/blob/34ad0469bacd3ef7076c5f753e49b4f0407f3d97/providers/sms-central/jest.config.js#L5-L8